src/moves.c

```diff
@@ -135,7 +135,7 @@ void InitPokemonMove(Move *move, u16 moveID)
     move->ginseng = 0;
 }
 
-void sub_8092AA8(Move *move, u16 moveID)
+void InitPokemonMoveOrNullObject(Move *move, u16 moveID)
```

Comment:
Not exactly sure on this name to be honest. It's not null. It's just the moveFlags is zeroed.

Reply:
It's basically an empty move object. The game will check for the move existing and find that it doesn't exist.